src/rr/core/GameManager.java

```diff
@@ -0,0 +1,72 @@
+///*
```

Comment:
Will you ever need this file?

If we are just completely commenting it out we can also delete it. Git will keep a history incase you ever want it back regardless.